common/src/macros.rs

```diff
@@ -22,3 +22,56 @@ macro_rules! optify {
         }
     };
 }
+
+/// Convenience macro to easily export items for wasm
+#[macro_export]
+macro_rules! if_wasm {
```

Comment:
these macros have been extremely nice to group multiple cfg-ed items together in a readable way

Reply:
Another option that doesn't use macros and kind of does the same thing as `MaybeSend` is to use a crate I published a while back called `is_not` (dumb name, I know), but it publishes two cfgs that are `#[is]` and `#[not]` that either return everything untouched or nothing. The idea is to use other configs to get those to export so you could export a wasm macro like

```
#[cfg(all(target_family = "wasm", target_os = "unknown"))]
pub use is_not::is as wasm;
#[cfg(not(all(target_family = "wasm", target_os = "unknown")))]
pub use is_not::not as wasm;
```

and the compiler resolves them the same way and it gets rid of the extra braces.

`is_not` has no dependencies (and should probably be a 1.0 release)

Reply:
another benefit is that these just stack so you can do things like

```rust
#[wasm]
#[d14n]
pub fn ...
```